.github/workflows/wsl2.yml

```diff
@@ -0,0 +1,69 @@
+name: WSL2
+
+# spell-checker:ignore nextest noprofile norc
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+# End the current execution if there is a new changeset in the PR.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  test:
+    name: Test
+    runs-on: ${{ matrix.job.os }}
+    timeout-minutes: 45
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - { os: windows-latest, distribution: Ubuntu-24.04, features: feat_os_unix}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+    - name: Install WSL2
+      uses: Vampire/setup-wsl@3b46b44374d5d0ae94654c45d114a3ed7a0e07a8 # v5.0.1
```

Comment:
would it be possible to use the version here?
I am not sure updatebot/renovatebot can deal with checksum